src/components/Congress/RepealLegislationButton/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import Button from '../../Button/Button';
+import CongressRepealLegislationModal from '../../Modals/CongressRepealLegislationModal';
+import { blockchainSelectors, congressSelectors } from '../../../redux/selectors';
+
+export default function RepealLegislationButton({ tier, index }) {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const handleModalOpen = () => setIsModalOpen(!isModalOpen);
+
+  // requires parent to dispatch getMembers action
+  const members = useSelector(congressSelectors.members);
+  const user = useSelector(blockchainSelectors.userWalletAddressSelector);
+
+  const userIsMember = members.map((m) => m.toString()).includes(user);
```

Comment:
```suggestion
  const userIsMember = members.includes(new TextEncoder().encode(user));
```

Reply:
Why?

Reply:
Also, it wouldn't even work.

Reply:
I thought it would be more efficient, but you're right